Name: "Palm" Carrier String
Version: 1.3.1-1
Author: Jason Robitaille
Description: Custom carrier string of "Palm" for the launcher.

--- .orig/usr/lib/luna/system/luna-systemui/app/controllers/bar-assistant.js
+++ /usr/lib/luna/system/luna-systemui/app/controllers/bar-assistant.js
@@ -457,7 +457,7 @@
 				$('rssi').className = "flight-mode";
 			}
 			else {
-				this.carrierText = $L("Phone offline");
+				this.carrierText = "Palm";
 				$('carrier').innerHTML = this.carrierText;
 			}							
 			$('roaming').hide();
@@ -480,7 +480,7 @@
 	this.phoneInLimitedService = false;	
 	switch (payload.extended.state) {
 		case 'service':	
-			this.carrierText = $L(payload.extended.networkName.escapeHTML());
+			this.carrierText = "Palm";
 			$('carrier').innerHTML = this.carrierText;	   	
 			if(payload.extended.registration == 'home') {			
 				$('roaming').hide();
@@ -808,7 +808,7 @@
 				this.updateAirplaneModeProgress('phone');
 			}
 			else {
-				this.carrierText = $L("Phone offline");
+				this.carrierText = "Palm";
 				$('carrier').innerHTML = this.carrierText;
 			}
 			$('datanet').className = 'off';
@@ -829,7 +829,7 @@
 		switch(networkMsg.state) {
 			case 'service':
 				this.phoneService = "service";
-				this.carrierText = $L(networkMsg.networkName.escapeHTML());
+				this.carrierText = "Palm";
 				$('carrier').innerHTML = this.carrierText;
 				this.simlocked = false;
 				
